lib/urbanopt/geojson.rb

```diff
@@ -50,14 +54,322 @@ def measures_dir
       def files_dir
         return nil
       end
-      
+
       # Doc templates are common files like copyright files which are used to update measures and other code
       # Doc templates will only be applied to measures in the current repository
       # Return the absolute path of the doc templates dir or nil if there is none
       def doc_templates_dir
         return File.absolute_path(File.join(@root_dir, 'doc_templates'))
       end
-      
+
+
+      def is_shadowed(building_points, other_building_points, origin_lat_lon)
```

Comment:
Should these be in Helpers?  How do you decide what is a helper or not?  Would be good to put some comments in each module to describe what kind of stuff goes in the module.